### ISSUE TITLE
CacheFragmentStatePagerAdapter is missing on maven

### ISSUE THREAD

**Issue by AndroidRio** [ksoichiro/Android-ObservableScrollView#54]
When I set the dependency to 
```
compile 'com.github.ksoichiro:android-observablescrollview:1.4.0'
```

The exploded aar doesnt contain the class CacheFragmentStatePagerAdapter.

**Comment by ksoichiro** [comment]
Sorry, it's in 1.5.0-SNAPSHOT and I haven't released it yet.
I'll do it soon.

**Comment by AndroidRio** [comment]
Oh ok. Thanks for the answer!

**Comment by ksoichiro** [comment]
I've released 1.5.0 and it's synced to Maven Central.

**Comment by AndroidRio** [comment]
Thanks!
